roles/cdi/templates/cdi-controller-deployment.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: cdi-deployment
+spec:
+  selector:
+    matchLabels:
+      app: cdi-controller
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cdi-controller
+    spec:
+      containers:
+      - name: cdi-controller
+        image: jcoperh/import-controller # need to tag this in docker hub
```

Comment:
please add the image tag here and as a top-level variable in defaults/main,yml with appropriate documentation.